src/Redux/configStore.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
```

Comment:
- [ ] As I pointed out in the previous comments, Kindly make sure you fix your linters error instead of disabling them. This is for best practices reasons and clean code